diff --git a/src/main/java/com/tek/ticket/controller/JobDescriptionController.java b/src/main/java/com/tek/ticket/controller/JobDescriptionController.java
index 0e4cc82..0bef000 100644
--- a/src/main/java/com/tek/ticket/controller/JobDescriptionController.java
+++ b/src/main/java/com/tek/ticket/controller/JobDescriptionController.java
@@ -53,6 +53,7 @@ public class JobDescriptionController {
 	 @GetMapping("/gitinfo")
 	    public ResponseEntity<Map<String, Object>> getGitInfo() throws IOException {
 	ClassPathResource classPathResource = new ClassPathResource("git.properties");
+	System.out.println("Hello");
 	        Properties properties = new Properties();
 	        properties.load(classPathResource.getInputStream());
 	        Map<String, Object> gitInfo = new HashMap<>();
